gitpod-io/website
# Rephrase reason why user should use the latest version of JetBrains Gateway

## Description

Rephrase reason why user should use the latest version of JetBrains Gateway.

## Related Issue(s)

Patches #1959 ([See discussion](https://github.com/gitpod-io/website/pull/1959#discussion_r862631647))

## How to test

- Visit the [Troubleshooting page in this Preview Environment](https://deploy-preview-1981--gitpod-kumquat.netlify.app/docs/troubleshooting#gitpod-logs-in-jetbrains-client).
- Check if the content from "Gitpod logs in JetBrains Gateway" and "Gitpod logs in JetBrains Client" sections are fine.

## Release Notes

```release-note
NONE
```

## Documentation

None.

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1981"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
